packages/boundless-popover/index.js

```diff
@@ -411,16 +417,34 @@ export default class Popover extends React.PureComponent {
         this.$caret.style[longitudinal ? 'top' : 'left'] = '0px';
     }
 
+    matchAnchorWidth = () => {
+      if (!this.props.matchAnchorWidth) return;
+
+      const anchor = this.props.anchor instanceof HTMLElement
```

Comment:
Just learned this recently, but `findDOMNode` is actually a no-op if called on a DOM node, so you can remove this ternary and always call it.